bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -126,7 +130,7 @@ class UserProfile(UpdateView):
     """
 
     template_name = "recordtransfer/profile.html"
-    paginate_by = 10
+    paginate_by = 2
```

Comment:
Go ahead and put this back up to 10, looks like it's working well.

Reply:
Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/280/commits/968f85ca9b89d347b94ae80188356f8875c33af6